### DISCUSSION TITLE
Adding New Customized Canvas

### DISCUSSION THREAD

**Discussion by diderich** [Leantime/leantime#996] [General] [answered]
What is the best/simplest way to create a new canvas and make it available like the Simple Canvas and Full Lean Canvas?

I am knowledgeable in php programming and creating tpl templates.

I am interested in adding the Lightweight Business Canvas model and Detailed Business Model Canvas from my book **Design Thinking for Strategy** (https://inov.at/dts-sn).

![lbm](https://user-images.githubusercontent.com/10230144/192148946-8b86645e-69bc-486b-9024-3982326a0927.jpg)

![dbm](https://user-images.githubusercontent.com/10230144/192148958-67f01242-86b6-438f-9483-dedd9c6240ac.jpg)

**Comment by marcelfolaron** [comment, marked as answer]
At this point the easiest would probably be to copy the entire "leancanvas"
module and adjust box fields as well as the html.
I'd love to have a more flexible solution to allow any type of canvas (SWOT
analysis, business model canvas, empathy map) etc. but that might take some
time to build.

Sincerely,
Marcel
